From 0000000000000000000000000000000000000000 Mon Jun 17 00:00:00 2024
From: Mahesh <mahakagi@amazon.com>
Date: Mon, 17 Jun 2024 20:17:20 -0700
Subject: [PATCH] --EKS-PRIVATE-- Bump go directive for tag 1.5.1 to fix go dependency download

Description:
This patch addresses the issue in cni-plugins-1-32-presubmit.
The upstream tag 1.5.1 specifies a go directive version of 1.20, which causes go mod vendor to fail while downloading dependencies.
To resolve this, the go directive is updated to 1.21 which has been verified after running 'go mod tidy'.
While we are using same Go version 1.22, consistent with the upstream, the upstream build uses 1.22-alpine. Our build process does not use Alpine, making this the difference in causing our build to fail.

---
 go.mod | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/go.mod b/go.mod
index 7445d206..7016423f 100644
--- a/go.mod
+++ b/go.mod
@@ -1,6 +1,6 @@
 module github.com/containernetworking/plugins
 
-go 1.20
+go 1.21
 
 require (
 	github.com/Microsoft/hcsshim v0.12.3
